src/main/kotlin/com/ajax/cryptocurrency/bpp/ScheduledBackgroundJobBeanPostProcessor.kt

```diff
@@ -44,6 +44,7 @@ class ScheduledBackgroundJobBeanPostProcessor(
         return bean
     }
 
+    @Suppress("UnusedPrivateMember")
     @PreDestroy
     private fun shutdown() {
```

Comment:
you can use internal instead of private and omit suppress annotation

Reply:
done